news_update.php

```diff
@@ -69,17 +69,15 @@
 
 echo '<div class="space-20"></div>';
 echo '<div class="alert alert-success">';
-echo lang_get( 'operation_successful' ) ?>
-
-echo '<br />';
+echo lang_get( 'operation_successful' );
+echo '<br /></div>';
 
 print_button( 'news_edit_page.php?news_id=' . $f_news_id . '&action=edit', lang_get( 'edit_link' ) );
+echo '&nbsp;&nbsp;&nbsp;&nbsp;';
```

Comment:
@atrol, please read the whole license at http://support.wrapbootstrap.com/knowledge_base/topics/usage-licenses
The item here is not Mantis, but Ace Admin template, and if it was based on a GPL code, it would be under GPL license, but so far we know it is not.
We have a completely different situation here, Ace Admin template (which itself might be under GPL, if it had a GPL basis) is non-exclusively and non-transferably incorporated into MantisBT according to the license preamble.